class-tgm-plugin-activation.php

```diff
@@ -758,17 +775,17 @@ public function notices() {
 					$install_link = true; // We need to display the 'install' action link.
 					$install_link_count++; // Increment the install link count.
 					if ( current_user_can( 'install_plugins' ) ) {
-						if ( isset( $plugin['required'] ) && $plugin['required'] ) {
-							$message['notice_can_install_required'][] = $plugin['name'];
+						if ( true === $plugin['required'] ) {
```

Comment:
How about if the user has registered slightly incorrect as a string:

`required => 'true',`

? We should allow a little leeway for 2.5, and then make it stricter for 3.0.